test/hoss/HOSS_Regression.ipynb

```diff
@@ -146,9 +147,13 @@
     "                             'bounds_collection': Collection(id='C1245618475-EEDTEST'),\n",
     "                             'bounds_granule_id': 'G1255863984-EEDTEST'}\n",
     "\n",
+    "hoss_prod_information = {'collection': Collection(id='C1996546500-GHRC_DAAC'),\n",
+    "                         'granule_id': 'G2040739631-GHRC_DAAC'}\n",
```

Comment:
This is exactly the same granule as used in UAT, so the same reference files are valid.